Polygon/include/CGAL/Polygon_2.h

```diff
@@ -472,28 +474,34 @@ class Polygon_2 {
     const Container_P& container() const
       { return d_container; }
 
+    /// Returns a reference to the sequence of vertices of the polygon.
     Container_P& container()
       { return d_container; }
 
+    /// Returns an iterator to the first vertex of the polygon.
     typename Container_P::iterator begin()
     {
        return container().begin();
     }
-
+    /// Returns an iterator to the element after the lastvertex of the polygon.
```

Comment:
Is that one purpose that you spell "lastvertex" instead of "last vertex"?

Reply:
No, it's a typo